### PR TITLE
sentry: port plugin to new composability API and dev-utils

### PR DESCRIPTION
**Pull request by Rugvip**
Ports the sentry plugin to use the new dev-utils and composability API

Not intended to be merged yet

### CODE REVIEW

File: plugins/sentry/dev/index.tsx [hunk 1]
```diff
@@ -56,52 +64,44 @@ createDevApp()
...
+            <EntityGridItem xs={12} md={6} entity={entity('empty')}>
+              <EntitySentryCard />
+            </EntityGridItem>
+            <EntityGridItem xs={12} md={6} entity={entity('never')}>
+              <EntitySentryCard />
+            </EntityGridItem>
+            <EntityGridItem xs={12} md={6} entity={entity('with-values')}>
+              <EntitySentryCard />
+            </EntityGridItem>
+            <EntityGridItem xs={12} entity={entity(undefined)}>
```
Comment (freben) [inline plugins/sentry/dev/index.tsx:99]: Well thus one will get undefined for name as well :)

File: plugins/sentry/src/extensions.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  createRoutableExtension,
+} from '@backstage/core';
+import { useEntity } from '@backstage/plugin-catalog';
+import { plugin, rootRouteRef } from './plugin';
+
+export const EntitySentryContent = plugin.provide(
+  createRoutableExtension({
+    mountPoint: rootRouteRef,
+    component: () =>
+      import('./components/SentryIssuesWidget').then(
```
Comment (freben) [inline plugins/sentry/src/extensions.tsx:29]: Hm so, this is a non-lazy, page-wrapped widget? 🤔

Comment (freben) [reply plugins/sentry/src/extensions.tsx:29]: Oh I see, it's just confusingly named Page, maybe rename that then

**Review by freben — APPROVED** [review]

File: plugins/sentry/src/extensions.tsx (hunk 2 above)
Comment (Rugvip) [reply plugins/sentry/src/extensions.tsx:29]: It's a page, same content as the old `Router` component

**Comment by Rugvip** [comment]
Added a changeset so that we could actually merge this now. Thinking next release though

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**